src/components/MainContainer/MainContainer.js

```diff
@@ -110,17 +113,15 @@ class Header extends Component {
         return (
             <div>
                 <div className="jumbotron">
-                    <h4 className="mainHeading">
-                        <strong>netflix</strong>roulette
-                    </h4>
-                    <h1 className="display-4 searchMovies">Find your Movie</h1>
+                    <Header />
                     <Search onKeyHandle={this.onKeyHandle} resultsQuery={this.resultsQuery}/>
                     <SortSearch sortTitles={this.sortTitles} genresActive={this.state.genresActive} titleActive={this.state.titleActive} sortGenres={this.sortGenres}/>      
                 </div>
                 <SortByCategory count={data.length} sortRelease={this.sortRelease} releaseActive={this.state.releaseActive} ratingActive={this.state.ratingActive} sortRating={this.sortRating} />              
-                <MoviesList data={data} sortTitles={this.sortTitles} />                        
+                <MoviesList data={data} sortTitles={this.sortTitles} /> 
```

Comment:
please follow https://github.com/airbnb/javascript/tree/master/react#props for props 
when you have 2+  props please break the line

Reply:
Updated